src/Sulu/Bundle/MediaBundle/Content/Types/SingleMediaSelection.php

```diff
@@ -66,7 +74,10 @@ public function getContentData(PropertyInterface $property): ?Media
             return null;
         }
 
-        if ($this->securityChecker
+        $webspace = $this->requestAnalyzer->getWebspace();
+
+        if ($webspace->hasWebsiteSecurity()
```

Comment:
Think in some cases the `$webspace` can here be also null so we need to check here for:

```suggestion
        if ($webspace
            && $webspace->hasWebsiteSecurity()
```